web/__init__.py

```diff
@@ -1,23 +1,43 @@
+import os
+import re
 import json
+import tempfile
 import logging as log
 
-from flask import Flask, request, render_template, Response
+from secrets import token_urlsafe as token
+
+from flask import Flask, flash, request, redirect, url_for, render_template, Response, session
 from flask_sock import Sock
+from werkzeug.utils import secure_filename
+
+import libflagship.httpapi
+import libflagship.logincache
 
 from libflagship.pppp import P2PSubCmdType, FileTransfer
 from libflagship.ppppapi import FileUploadInfo, PPPPError
 
 
 from web.lib.service import ServiceManager
 
+from user_agents import parse
```

Comment:
With a name as generic as `parse`, that we use only 1 place, we should `import user_agents`, and call `user_agents.parse()` where needed. That will be much easier to follow.

Also, the rough order of imports should be:

 - python builtins
 - modules from pip dependencies
 - modules from this package